docs/typescript/overview.md

```diff
@@ -22,21 +20,21 @@ With a Vite-based setup, the dev server and the bundler are transpilation-only a
 
 - Vue CLI also provides TypeScript support, but is no longer recommended. See [notes below](#note-on-vue-cli-and-ts-loader).
 
-### IDE Support {#ide-support}
+### IDE Support
 
 - [Visual Studio Code](https://code.visualstudio.com/) (VSCode) is strongly recommended for its great out-of-the-box support for TypeScript.
 
   - [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) is the official VSCode extension that provides TypeScript support inside Vue SFCs, along with many other great features.
 
-    :::tip
-    Volar replaces [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur), our previous official VSCode extension for Vue 2. If you have Vetur currently installed, make sure to disable it in Vue 3 projects.
-    :::
+    <box type="info">
```

Comment:
Same as above

Reply:
I noticed that the links in the box doesn't work if there isn't an empty line between <box> and the content. Not sure if its the same for you?

Reply:
I just noticed it doesn't work as well, may I know if you managed to find a workaround besides having that empty line?